postUserAuthentication/createOrgOnSignupWorkflow.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  onPostAuthenticationEvent,
+  WorkflowSettings,
+  WorkflowTrigger,
+  createKindeAPI,
+} from "@kinde/infrastructure";
+
```

Comment:
@Upatric could you also add this entry to the `readme.md`, you'll see there is a list already